src/lib/prisma.ts

```
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client';
+
+declare global {
+  // Allow global `var` declarations
+  // eslint-disable-next-line no-var
+  var prisma: PrismaClient | undefined;
+}
+
+// Prevent multiple instances of Prisma Client in development
+const prisma = globalThis.prisma || new PrismaClient();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)